### PR TITLE
[Notifier] fix test

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php [hunk 1]
```diff
@@ -64,7 +64,10 @@ public function testValidSignature(string $message)
 
         $time = time();
 
-        $lastResponse = new MockResponse();
+        $data = json_encode([
+            'ids' => ['0611223344'],
```
Comment (xabbuh) [inline src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php]: @OskarStark @antiseptikk Do OvhCloud ids have a special format that would be worth to follow here?

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php]: Hmm based on https://docs.ovh.com/gb/en/sms/send_sms_messages_via_url_-_http2sms

```json
{"status":100,"creditLeft":"1987","SmsIds":["10867690"]}
```

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php]: But based on https://docs.ovh.com/gb/en/sms/send_sms_with_ovhcloud_api_in_php/

it is:
```json
Array
(
    [totalCreditsRemoved] => 1
    [invalidReceivers] => Array
        (
        )

    [ids] => Array
        (
            [0] => 26929925
        )

    [validReceivers] => Array
        (
            [0] => +3360000000
        )

)
```

which I think is correct

File: src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php [hunk 2]
```diff
@@ -64,7 +64,17 @@ public function testValidSignature(string $message)
...
 
-        $lastResponse = new MockResponse();
+        $data = json_encode([
+            'totalCreditsRemoved' => '1',
+            'invalidReceivers' => [],
+            'ids' => [
+                '26929925',
+            ],
+            'validReceivers' => [
+                '+3360000000',
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php]: I think we should match this at least with the number used in SmsMessage above

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php (hunk 2 above)
Comment (xabbuh) [reply src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php]: fixed

**Review by OskarStark — APPROVED** [review]

**Pull request merged**